### PR TITLE
Added gl_compatibility as an option to the project creation screen

### PR DESCRIPTION
**Pull request by clayjohn**
This PR adds the gl_compatibility renderer to the options on project creation. As discussed with Akien on RocketChat, we are doing a very simple option menu for now, but in later releases we will expand this option menu further to allow choosing more than just a default renderer

This also saves the last selected renderer as the default renderer to be checked off next time you start a project.

Screenshot from 2022-12-13 14-10-57
Screenshot from 2022-12-13 14-10-53

### CODE REVIEW

**Comment by YuriSizov** [comment]
Should we explicitly state that this is for OpenGL, because people are going to look for that?

**Comment by clayjohn** [comment]
> Should we explicitly state that this is for OpenGL, because people are going to look for that?

Yes, good idea, I'll update the text

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

File: editor/project_manager.cpp [hunk 1]
```diff
@@ -426,6 +427,35 @@ class ProjectDialog : public ConfirmationDialog {
...
+		} else if (renderer_type == "mobile") {
+			renderer_info->set_text(
+					String::utf8("•  ") + TTR("Supports desktop + mobile platforms.") +
+					String::utf8("\n•  ") + TTR("Less advanced 3D graphics.") +
+					String::utf8("\n•  ") + TTR("Less scalable for complex scenes.") +
+					String::utf8("\n•  ") + TTR("Uses RenderingDevice backend.") +
+					String::utf8("\n•  ") + TTR("Fast rendering of simple scenes."));
+		} else if (renderer_type == "gl_compatibility") {
+			renderer_info->set_text(
+					String::utf8("•  ") + TTR("Supports desktop, mobile, + web platforms.") +
```
Comment (Anutrix) [inline editor/project_manager.cpp:449]: Any reason `, +` was used instead of `and`?

Comment (clayjohn) [reply editor/project_manager.cpp:449]: To match the message used by the mobile renderer

Comment (akien-mga) [reply editor/project_manager.cpp:449]: I think the point is that `, +` is broken syntax. The comma should be removed.

Comment (clayjohn) [reply editor/project_manager.cpp:449]: Oh, I guess so. Oxford comma doesn't make sense with + instead of and

**Comment by oliver20033549** [comment]
some features are still missing.

**Comment by aragubas** [comment]
Amazing!

**Pull request merged**